schematics/ng-componize/htmlParser.ts

```diff
@@ -0,0 +1,167 @@
+import { ClassDeclaration, Project, SourceFile } from 'ts-morph';
+
+// rename ??
+export interface FoundItem {
```

Comment:
Why not create multiple files instead putting it all in the same file ? it will make the html parser easier to read